lib/elixir/lib/process.ex

```diff
@@ -399,8 +398,8 @@ defmodule Process do
 
   Inlined by the compiler.
   """
-  @spec demonitor(reference) :: true
   @spec demonitor(reference, options :: [:flush | :info]) :: boolean
+  @spec demonitor(reference, []) :: true
```

Comment:
I would go with `[:flush | :info, ...] :: boolean` and `[] :: true`. For me is not confusing as the latter is an addition to the spec that makes it even more clear.
But it's up to you guys